### PR TITLE
EnglishPremierLeagueCheatSheet: Updated the participated teams. 

### PR DESCRIPTION
**Pull request by ThePinger**
## Description of new Instant Answer, or changes

Removed the relegated teams after the 2016 season and added the promoted teams from Championship and updated the current title holder.

## People to notify

@gautamkrishnar  @moollaza 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable (for "More at {source_name}" link)
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/epl_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/epl.json [hunk 1]
```diff
@@ -18,30 +18,34 @@
...
             "key": "ARS",
             "val": "Arsenal (The Gunners)",
             "link": "http://www.arsenal.com/home"
-        }, {
+        },{
             "key": "BOU",
             "val": "Bournemouth (The Cherries)",
             "link": "http://www.afcb.co.uk"
-        }, {
+        },{
```
Comment (GPhantom) [inline share/goodie/cheat_sheets/json/epl.json]: @ThePinger Please add a space after the comma. Apply this for the rest of the code.

Comment (ThePinger) [inline share/goodie/cheat_sheets/json/epl.json]: Done

File: share/goodie/cheat_sheets/json/epl.json [hunk 2]
```diff
@@ -103,7 +103,7 @@
         }],
         "Glossary": [{
                 "key": "The Title",
-                "val": "The club that finishes at the top of the league wins the title.  In 2014, Chelsea won the Premier League Title"
+                "val": "The club that finishes at the top of the league wins the title.  In 2016, Leicester City won the Premier League Title"
```
Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/epl.json]: Nit: There is multiple spaces between `.` and `In`. Please fix it. 🙂

Comment (ThePinger) [inline share/goodie/cheat_sheets/json/epl.json]: Do you mean the alphabetic order by the abbreviation or the team full name ? @ManrajGrover

Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/epl.json]: @ThePinger, I feel they should be sorted based on `key`.

Comment (ThePinger) [inline share/goodie/cheat_sheets/json/epl.json]: @ManrajGrover  DONE

**Comment by ManrajGrover** [comment]
@pjhampton Looks like this build is failing due to unrelated test file `Passphrase` failing on a test case.

**Comment by pjhampton** [comment]
@ManrajGrover Thanks! I've restarted the CI build. It might be a deeper rooted issue though 👍

**Comment by sahildua2305** [comment]
Thanks, @ManrajGrover and @pjhampton for looking into the build issue.

**Pull request merged**